city_scrapers/spiders/chi_police_retirement.py

```diff
@@ -0,0 +1,129 @@
+import re
+from datetime import datetime
+
+from city_scrapers_core.constants import BOARD
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class ChiPoliceRetirementSpider(CityScrapersSpider):
+    name = "chi_police_retirement"
+    agency = "Policemen's Annuity and Benefit Fund of Chicago"
+    timezone = "America/Chicago"
+    allowed_domains = ["www.chipabf.org"]
+    start_urls = ["http://www.chipabf.org/ChicagoPolicePension/MonthlyMeetings.html"]
+    TAG_RE = re.compile(r'<[^>]+>')
+
+    def parse(self, response):
+        year = self._parse_year(response)
+        date_table = response.xpath('//*[@id="content0"]/div[3]/table')
+        date_items = date_table.extract()[0].split('<tr>')
+
+        skip_first_item = True
+        for item in date_items:
+            if skip_first_item:
+                skip_first_item = False
+                continue
+
+            meeting = Meeting(
+                title=self._parse_title(response),
+                description=self._parse_description(response),
+                classification=self._parse_classification(response),
+                start=self._parse_start(item, year),
+                time_notes=self._parse_time_notes(),
+                end=self._parse_end(),
+                all_day=False,
+                location=self._parse_location(response),
+                source=self._parse_source(response),
+                links=self._parse_links(item)
+            )
+
+            # replace before this
+            meeting["status"] = self._get_status(meeting)
+            meeting["id"] = self._get_id(meeting)
+
+            yield meeting
+
+    def _parse_title(self, item):
```

Comment:
It looks like there are two types of meeting that can be parsed here: Investment Committee meetings and Board meetings. You should be able to check if the first cell in the row has "Investment" in it, and return "Investment Committee" if so, otherwise return "Retirement Board".

Since we know what the two categories are going to be it's easier to return strings based on conditions than returning the parsed string.